dspace-spring-rest/src/main/java/org/dspace/app/rest/converter/EPersonConverter.java

```diff
@@ -11,8 +11,8 @@
 import java.util.List;
 
 import org.apache.log4j.Logger;
-import org.dspace.app.rest.model.GroupRest;
 import org.dspace.app.rest.model.EPersonRest;
+import org.dspace.app.rest.model.GroupRest;
```

Comment:
I know that we have an open conversation about setting code style to avoid such kind of issue... but can you exclude from the PR the file that only have fix to the import? this should be done in a single dedicated pr when we have set our codestyle. This PR doesn't really need to touch more than 5-10 files I guess